src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="txt-area">0</div>
+        <div className="input-area">
+          <div className="row">
+            <div>Ac</div>
+            <div>+/-</div>
+            <div>%</div>
+            <div className="orange">÷</div>
+          </div>
+          <div className="row">
+            <div>7</div>
+            <div>8</div>
+            <div>9</div>
+            <div className="orange">x</div>
+          </div>
+          <div className="row">
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+            <div className="orange">-</div>
+          </div>
+          <div className="row">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div className="orange">+</div>
+          </div>
+          <div className="row">
+            <div className="btn-zero">0</div>
+            <div>.</div>
+            <div className="orange">=</div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
- [ ] In the upcoming project you will design the calculator on a small area as you have a `nav` bar and some content so please make the calculator visible in a small space. Your calculator is now covering the full height and width of the browser. Use the image below for improvement.
### Design
![sssss](https://user-images.githubusercontent.com/81727988/188870041-6a452651-c773-49c7-a496-5698dff500d1.png)